pyega3/pyega3.py

```diff
@@ -38,606 +33,707 @@
+                                                                    res['fileId'],
+                                                                    res['displayFileName'],
+                                                                    genomic_range_args)
+
+                    self.download_file_retry(
+                        credentials, res['fileId'], res['displayFileName'], res['fileName'], res['fileSize'],
+                        res['unencryptedChecksum'],
+                        num_connections, key, output_file, genomic_range_args, max_retries, retry_wait)
+            except Exception as ex:
+                logging.exception(ex)
+
+    @staticmethod
+    def print_debug_info(url, reply_json, *args):
+        logging.debug("Request URL : {}".format(url))
+        if reply_json is not None:
+            logging.debug("Response    :\n %.1200s" % json.dumps(reply_json, indent=4))
+
+        for arg in args:
+            logging.debug(arg)
+
+    @staticmethod
+    def create_parser():
+        parser = argparse.ArgumentParser(description="Download from EMBL EBI's EGA (European Genome-phenome Archive)")
+        parser.add_argument("-d", "--debug", action="store_true", help="Extra debugging messages")
+        parser.add_argument("-cf", "--config-file", dest='config_file',
+                            help='JSON file containing credentials/config e.g.{"username":"user1","password":"toor"}')
+        parser.add_argument("-sf", "--server-file", dest='server_file',
+                            help='JSON file containing server config e.g.{"url_auth":"aai url","url_api":"api url", '
+                                 '"url_api_ticket":"htsget url", "client_secret":"client secret"}')
+        parser.add_argument("-c", "--connections", type=int, default=1,
+                            help="Download using specified number of connections")
+        parser.add_argument("-t", "--test", action="store_true", help="Test user activated")
+
+        subparsers = parser.add_subparsers(dest="subcommand", help="subcommands")
+
+        subparsers.add_parser("datasets", help="List authorized datasets")
+
+        parser_dsinfo = subparsers.add_parser("files", help="List files in a specified dataset")
+        parser_dsinfo.add_argument("identifier", help="Dataset ID (e.g. EGAD00000000001)")
+
+        parser_fetch = subparsers.add_parser("fetch", help="Fetch a dataset or file")
+        parser_fetch.add_argument("identifier", help="Id for dataset (e.g. EGAD00000000001) "
+                                                     "or file (e.g. EGAF12345678901)")
+
+        parser_clean = subparsers.add_parser("clean", help="Deletes those temporary files "
+                                                           "which were created during the download process.")
+        parser_clean.add_argument("--dir", "-d", type=str, default=None,
+                                  help="The directory in which the temporary files should be deleted. "
```

Comment:
Suggest help="The directory from which the temporary files should be deleted."

Reply:
Done.